freeipa/src/main/java/com/sequenceiq/freeipa/converter/telemetry/TelemetryConverter.java

```diff
@@ -73,6 +74,12 @@ private Logging createLoggingFromRequest(LoggingRequest loggingRequest) {
                 adlsGen2Params.setSecure(adlsGen2FromRequest.isSecure());
                 adlsGen2Params.setManagedIdentity(adlsGen2FromRequest.getManagedIdentity());
                 logging.setAdlsGen2(adlsGen2Params);
+            } else if (loggingRequest.getCloudwatch() != null) {
+                CloudwatchParams cloudwatchRequest = loggingRequest.getCloudwatch();
+                CloudwatchParams cloudwatchParams = new CloudwatchParams();
```

Comment:
Like in `TelemetryApiConverter`, this could be refactored using a copy constructor or static factory method.